TestBenches/FileReadUtility.hh

```diff
@@ -138,7 +138,7 @@ unsigned int compareMemWithFile(const MemType& memory, std::ifstream& fout,
   for (int i = 0; i < memory_ref.getEntries(ievt); ++i) {
 
     // Maximum processing steps per event is kMaxProc
-    if (i >= kMaxProc) {
+    if (i >= kMaxProc-10) {
```

Comment:
What is this magic number?

Reply:
Whoops, this is a temporary hack from when I was having issues with truncation that I forgot to fix. I'll fix this now.

Reply:
When I look at this file in the AddIntegrationTest branch and the master branch, this hack isn't present. Also Github says that the file from this comment is Outdated. So I think this is already fixed in the AddIntegrationTest branch.

Reply:
ok, great. Not sure why it appeared in the diff earlier but maybe I did something odd.